package.json

```diff
@@ -29,7 +29,7 @@
   "scripts": {
     "start": "DEBUG=raneto node example/server.js",
     "start_win": "set DEBUG=raneto&&node example/server.js",
-    "test": "npm run lint & npm run compile & mocha --reporter spec test/*.js",
+    "test": "npm run lint && npm run compile && mocha --reporter spec test/*.js",
```

Comment:
@theRealWardo thanks for the PR
Can you provide more detail on this section?
We originally had double ampersands but switched to single because it worked on *NIX/Windows
Did you have issues with this on your platform?